example/build.gradle

```diff
@@ -39,13 +39,15 @@ android {
             minifyEnabled true
             proguardFiles getDefaultProguardFile('proguard-android.txt')
 
-            signingConfig signingConfigs.release
+            if (file(releaseKeyStoreFileName).exists() && file(releasePropFileName).exists()) {
+                signingConfig signingConfigs.release
```

Comment:
Thanks for explaining your reasoning! I think I'd prefer the following compromise:

- revert changes so that building a release build still fails fast;
- update the CONTRIBUTING file explaining how to build the project (assembling the release variant of the library itself; assembling the debug variant of the example app).

This seems to make most sense given the intended role of the CONTRIBUTING file. So yes, please revert the changes to the signing configs in this PR, then I will separately update the CONTRIBUTING info :)